cartographer/io/migrate_serialization_format_main.cc

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Make sure to remove this file again before merging.

Reply:
Good catch, thought this was automatically removed. Changed.